server/src/test/java/org/elasticsearch/action/admin/cluster/node/stats/NodeStatsTests.java

```diff
@@ -555,11 +558,26 @@ public static NodeStats createNodeStats() {
             List<ScriptContextStats> stats = new ArrayList<>(numContents);
             HashSet<String> contexts = new HashSet<>();
             for (int i = 0; i < numContents; i++) {
+                List<ScriptContextStats.TimeSeries> timeSeries = new ArrayList<>();
```

Comment:
I think this would be more readable if it was moved to a helper function, eg `randomTimeSeries()`, then it can be used below inline, without needing this weirdness with a length 2 list.

Reply:
Moved.